tests/unit/challenges/test_models.py

```diff
@@ -142,3 +166,68 @@ def test_get_start_date(self):
     def test_get_end_date(self):
         self.assertEqual(self.challenge_phase.end_date,
                          self.challenge_phase.get_end_date())
+
+
+class LeaderboardTestCase(BaseTestCase):
+
+    def setUp(self):
+        super(LeaderboardTestCase, self).setUp()
+        try:
+            os.makedirs('/tmp/evalai')
+        except OSError:
+            pass
+
+        with self.settings(MEDIA_ROOT='/tmp/evalai'):
+            self.leaderboard = Leaderboard.objects.create(schema=json.dumps({"hello": "world"}))
+
+    def tearDown(self):
+        shutil.rmtree('/tmp/evalai')
+
+    def test__str__(self):
+        instance_id = str(self.leaderboard.id)
+        self.assertEqual(instance_id,
+                         self.leaderboard.__str__())
+
+
+class ChallengePhaseSplitTestCase(BaseTestCase):
+
+    def setUp(self):
+        super(ChallengePhaseSplitTestCase, self).setUp()
+        try:
+            os.makedirs('/tmp/evalai')
+        except OSError:
+            pass
+
+        with self.settings(MEDIA_ROOT='/tmp/evalai'):
+            self.challenge_phase = ChallengePhase.objects.create(
+                name='Challenge Phase',
+                description='Description for Challenge Phase',
+                leaderboard_public=False,
+                is_public=False,
+                start_date=timezone.now() - timedelta(days=2),
+                end_date=timezone.now() + timedelta(days=1),
+                challenge=self.challenge,
+                test_annotation=SimpleUploadedFile('test_sample_file.txt',
+                                                   'Dummy file content', content_type='text/plain')
+            )
+
+        self.dataset_split = DatasetSplit.objects.create(name="Test Dataset Split", codename="test-split")
+
+        self.leaderboard = Leaderboard.objects.create(schema=json.dumps({'hello': 'world'}))
+
+        self.challenge_phase_split = ChallengePhaseSplit.objects.create(
+            dataset_split=self.dataset_split,
+            challenge_phase=self.challenge_phase,
+            leaderboard=self.leaderboard,
+            visibility=ChallengePhaseSplit.PUBLIC
+            )
+
+    def tearDown(self):
+        shutil.rmtree('/tmp/evalai')
+
+    def test__str__(self):
+        challenge_phase_name = self.challenge_phase_split.challenge_phase.name
+        dataset_split_name = self.challenge_phase_split.dataset_split.name
+        string = challenge_phase_name + " : " + dataset_split_name
```

Comment:
Here instead of joining the string, we can use format function like ```'{}:{}'.format(challenge_phase_name, dataset_split_name)```. Also can we please make the name more meaningful(string is kind of generic)